Fix pytest usage

> ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
> pytest: error: unrecognized arguments: -n

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -84,7 +84,6 @@ warn_unused_ignores = false
 
 
 [tool.pytest.ini_options]
-addopts = "-n auto"
 testpaths = [
     "tests"
 ]
